template.env

```diff
@@ -31,19 +39,25 @@ ALLIGN_ORIGIN=True
 # AVE_GAMMA=3
 # AVERAGE_METHOD=dtw
 
-
 # DOCKER
 DOCKER_IMG=skhatiri/aerialist
 DOCKER_TIMEOUT=1000
 # SIMULATION_TIMEOUT=970
 
 # k8s
-# ROS_KUBE_TEMPLATE=aerialist/resources/k8s/k8s-job-avoidance.yaml
+ROS_KUBE_TEMPLATE=aerialist/resources/k8s/k8s-job-avoidance-pattern.yaml
```

Comment:
keep your specific configuration defaults in a seperate file or comment (#) and describe them here